GooglePayment/src/main/java/com/braintreepayments/api/models/ReadyForGooglePayRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.braintreepayments.api.models;
+
+import com.braintreepayments.api.BraintreeFragment;
+import com.braintreepayments.api.GooglePayment;
+import com.braintreepayments.api.interfaces.BraintreeResponseListener;
+
+/**
+ * Optional parameters to use when checking whether Google Payment API is supported and set up on the device.
+ */
+public class ReadyForGooglePayRequest {
+
+    private boolean mExistingPaymentMethodRequired;
+
+    /**
+     * If set to true, then the {@link GooglePayment#isReadyToPay(BraintreeFragment, ReadyForGooglePayRequest, BraintreeResponseListener)}
+     * method will call the listener with true if the customer is ready to pay with one or more of your
+     * allowed card networks.
```

Comment:
This string confuses me. How is `existingPaymentMethodRequired` relevant to the text on line 15-17?

Reply:
We're basically wrapping a call to Google's `isReadyToPay` method. Previously, there was no way for merchants to set a property called `existingPaymentMethodRequired` to true, but now we're exposing this option.

In case it helps, this is how Google describes that property in their docs:
> If set to true, then the isReadyToPay() class method will return true if the current viewer is ready to pay with one or more payment methods specified in allowedPaymentMethods.

Reply:
The difference is: 

If `existingPaymentMethodRequired` is false, `isReadyToPay` will return true as long as the customer has Google Pay on their device. 

If `existingPaymentMethodRequired` is true, `isReadyToPay` will return true only if the customer has GPay on their device **and** they have a card in GPay that's in one of the merchant's allowed card networks. 

Totally open to suggestions for how to clarify this!

Reply:
I feel like it might be better to say "supported card networks" instead of "allowed card networks".

Reply:
Your explanation clarifies this 👍